src/services/tx/txSender.ts

```diff
@@ -22,35 +22,34 @@ import type { Web3Provider } from '@ethersproject/providers'
 import type { ContractTransaction } from 'ethers'
 import { ethers } from 'ethers'
 import type { SpendingLimitTxParams } from '@/components/tx/modals/TokenTransferModal/ReviewSpendingLimitTx'
-import { getSpendingLimitContract } from '@/services/contracts/spendingLimitContracts'
+import { getSpendingLimitContract, getSpendingLimitModuleAddress } from '@/services/contracts/spendingLimitContracts'
 import EthersAdapter from '@gnosis.pm/safe-ethers-lib'
 import { Errors, logError } from '@/services/exceptions'
 import { EMPTY_DATA } from '@gnosis.pm/safe-core-sdk/dist/src/utils/constants'
+import { useCallback } from 'react'
+import type { SpendingLimitState } from '@/store/spendingLimitsSlice'
+import { createAddDelegateTx, createResetAllowanceTx, createSetAllowanceTx } from '@/services/tx/spendingLimitParams'
+import { parseUnits } from '@ethersproject/units'
+import { currentMinutes } from '@/utils/date'
+import type { NewSpendingLimitData } from '@/components/settings/SpendingLimits/NewSpendingLimit/steps/ReviewSpendingLimit'
```

Comment:
Given that these dependencies are only for the spending limits, I would move the spending limit function into a separate module and re-export it here.

Reply:
I extracted the function into a new file which is also exported by the txSender.